app/views/newsletters/index.html.erb

```diff
@@ -31,7 +31,7 @@
                   card.with_image(src:
               cdn_link_url(newsletter.image),
                                   rounded: :top, classes:
-              'border-light bg-light border-top border-left border-right')
+              'border-light border-top border-left border-right')
```

Comment:
Didn't mean to leave the bg-light in here, just the border-light, so removed it.